lib/parliament/utils/config/initializers/sanitizer.rb

```diff
@@ -0,0 +1,4 @@
+Rails.application.config.after_initialize do
+  ActionView::Base.sanitized_allowed_tags += %w(table thead tbody tfoot tr th td iframe)
```

Comment:
Is there a way of limiting this in a certain context rather than changing it globally - I feel that globally adding iframes is a risk?